src/batch/executors/src/executor/insert.rs

```diff
@@ -274,10 +274,9 @@ mod tests {
     use risingwave_common::transaction::transaction_message::TxnMsg;
     use risingwave_common::types::StructType;
     use risingwave_dml::dml_manager::DmlManager;
-    use risingwave_storage::hummock::test_utils::*;
+    use risingwave_storage::hummock::test_utils::{ReadOptions, *};
```

Comment:
I wonder if the `use` of `ReadOptions` is included in `*`?

Reply:
It used to conflict with `use risingwave_storage::store::ReadOptions;`, but after it is removed, it should not have conflict.

I will remove it later.